.coveragerc

```diff
@@ -433,6 +431,11 @@ omit =
     homeassistant/components/weather/openweathermap.py
     homeassistant/components/weather/zamg.py
     homeassistant/components/zeroconf.py
+    homeassistant/components/zwave/__init__.py
+    homeassistant/components/zwave/const.py
```

Comment:
This file has 100% test coverage because there is no code, just constants. No need to add it here.